etna/analysis/feature_relevance/plots.py

```diff
@@ -81,7 +81,7 @@ def plot_feature_relevance(
     if segments is None:
         segments = sorted(ts.segments)
     border_value = None
-    features = list(set(ts.columns.get_level_values("feature")) - {"target"})
+    features = list(set(ts.features) - {"target"})
```

Comment:
Why not use `list.remove`? Asymptotically it will be the same as the current solution.

Reply:
This replacement was done without any thinking. Calling `list.remove` fails if there is not "target". I don't think that it is really important, but the behavior is slightly different.

Reply:
This slightly improves performance since we will need one pass instead of two. Here we work with a dataset, so `target` should be presented at any time by convention.